pkg/app/launcher/cmd/launcher/launcher.go

```diff
@@ -279,6 +332,7 @@ func (l *launcher) run(ctx context.Context, input cli.Input) error {
 			select {
 			case <-ticker.C:
 				// Don't return an error to continue piped execution.
+				l.enqueueNewCommands(ctx, input)
```

Comment:
I feel it's a bit overdo here. Since the command restart piped is not that much and it should be handled immediately on given, making a queue of commands for that is unneeded 🤔 Should we just list commands and restart it instead of sending them to the commandhandler? wdyt @knanao @Szarny

Reply:
I got your point and I feel it's better but it's up to you since there are already some implementations like this. 🙌